src/backend/base/langflow/schema/schema.py

```diff
@@ -41,77 +41,68 @@ class OutputValue(BaseModel):
 
 
 def get_type(payload):
-    result = LogType.UNKNOWN
     match payload:
         case Message():
-            result = LogType.MESSAGE
+            return LogType.MESSAGE
 
         case Data():
-            result = LogType.DATA
+            return LogType.DATA
 
         case dict():
-            result = LogType.OBJECT
+            return LogType.OBJECT
 
         case list() | DataFrame():
-            result = LogType.ARRAY
+            return LogType.ARRAY
 
         case str():
-            result = LogType.TEXT
+            return LogType.TEXT
 
-    if result == LogType.UNKNOWN and (
-        (payload and isinstance(payload, Generator))
-        or (isinstance(payload, Message) and isinstance(payload.text, Generator))
```

Comment:
Just wondering if there's not a logic bug (already existing) when `payload` is a `Message` and `payload.text` is a `Generator`. @ogabrielluiz ?

Reply:
yeah its possible there is an existing bug that caused this part of the code to never be run.. Thats why i simplified. If the intention is different, feel free to fix.